src/modules/scope.js

```diff
@@ -102,6 +102,13 @@ export default class Scope {
     window.removeEventListener('keydown', this.handleKeyDown);
   };
 
+  checkFocus = (target: HTMLElement) => {
```

Comment:
If the target is not found won't it create an infinite loop?
How to handle that?

Reply:
The check for target existing is on L:119 - I think that would prevent an infinite loop.